languagetool-language-modules/ga/src/main/resources/org/languagetool/rules/ga/grammar.xml

```diff
@@ -5411,6 +5411,22 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
             <short>Neamhchaighdeánach</short>
             <example correction="cá mhéad"><marker>cá mhéid</marker></example>
         </rule>
+        <rule id="AG_DEIREADH_AN_LAE" name="Ag deireadh an lae">
+            <antipattern>
+                <token>ag</token>
+                <token>deireadh</token>
+                <token>an</token>
+		<token>lae</token>
+            </antipattern>
+            <pattern>
+                <token>i</token>
+                <token>ndeireadh</token>
+		<token>na</token>
+		<token>dálai</token>
+            </pattern>
+            <message>Is é<suggestion>i ndeireadh na dála</suggestion> an nath ceart i nGaeilge.</message>
+            <example correction="i ndeireadh na dála">Ach <marker>i ndeireadh na dála</marker> rop sí lámh léi amach san fhuinneog.</example>
```

Comment:
`<marker>` does not contain the error, therefore cannot be matched by `<pattern>`